bundle/edu.gemini.ui.workspace/src/main/java/edu/gemini/ui/workspace/util/SimpleInternalFrame.java

```diff
@@ -320,7 +274,7 @@ public void setSelected(boolean newValue) {
      * @param bar     the panel's tool bar
      * @return the panel's built header area
      */
-    private JPanel buildHeader(JLabel label, JToolBar bar, Action helpAction, Icon helpIcon) {
+    private JPanel buildHeader(JLabel label, JToolBar bar, Action helpAction, Icon helpIcon, String hint) {

```

Comment:
Apart from `null` distaste and wondering if we should instead use an `Option<String>` here, with a `hintOption.foreach(help::setToolTipText)`,  :+1: